doc/releases/release-notes-template.rst

```diff
@@ -0,0 +1,269 @@
+:orphan:
+
+.. _zephyr_X.Y:
+
+Zephyr X.Y.0 (Working Draft)
+############################
+
+We are pleased to announce the release of Zephyr version X.Y.0.
+
+.. This section is to be populated by the release manager
+
+Major features and improvements in this release include:
+
+*
+*
+
+API Changes
+***********
+
+.. Please document all API additions, removals, deprecation and changes here and
+   not in the individual subsections
+
+New APIs
+========
+
+Removed APIs
+============
+
+Deprecated APIs
+===============
+
+Changed APIs
+============
+
+.. Includes both stable and unstable/experimental API changes
+
+Security Vulnerabilities
+************************
+
+Detailed changelog
+******************
+
+Kernel
+======
+
+Architectures
+=============
+
+* ARC
+
+* ARM
+
+* ARM64
+
+* MIPS
+
+* NIOS2
+
+* POSIX
+
+* RISC-V
+
+* SPARC
+
+* x86
+
+* Xtensa
+
+Bluetooth
+=========
+
+* Audio
```

Comment:
3.4 has a general heading first

Reply:
Yes, but that is usually not required. If we need it I can re-add it back later.